src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.Expander.cs

```diff
@@ -930,6 +930,13 @@ public override SyntaxNode VisitInvocationExpression(InvocationExpressionSyntax
                 ExpressionSyntax thisExpression,
                 IMethodSymbol reducedExtensionMethod)
             {
+                var originalMemberAccess = (MemberAccessExpressionSyntax)originalNode.Expression;
+                if (originalMemberAccess.GetParentConditionalAccessExpression() != null)
+                {
+                    // Bail out on extension method invocations in conditional access expression.
```

Comment:
Nit: If possible, tag the GH issue tracking this Todo work here, so that when someone looks at it in the future, they understand this is simply a workaround and not an intentional design.